cmsranking/static/TimeView.js

```diff
@@ -142,6 +133,10 @@ var TimeView = new function () {
             }
         }
 
+        // if the contest has just ended (or has just started) then the chart
+        // on the left is bad positioned: "simulate" window resizing to fix it
+        $(window).trigger('resize');
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ["$" is not defined.](https://www.codacy.com/app/s-maggiolo/cms/file/2192941803/issues/source?bid=3167167&fileBranchId=3167185#l138)